Sources/Kitura/RouterHTTPVerbs_generated.swift

```diff
@@ -19,567 +19,1539 @@
 extension Router {
     // MARK: All
 
+    /// Setup a set of one or more closures of the type `RouterHandler` that will be
+    /// invoked when any request comes to the server. If a path pattern is
+    /// specified, the handlers will only be invoked if the pattern is matched.
+    ///
+    /// - Parameter path: An optional String specifying the patern that needs to be
```

Comment:
Typo: patern -> pattern - and in the resulting places below where it has also been pasted